docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
-[Bitbucket](https://bitbucket.org) repositories, with
-[status checks](/guides/cloud/integrations/github#Status-checks) that block
-commits from being merged until your Cypress tests are green. Surface test
+Cypress Cloud integrates with GitHub, GitLab, and Bitbucket to provide
+rock-solid reliability by keeping failing code out of your main branch. Use
+[status checks](/guides/cloud/integrations/github#Status-checks) to block
+commits from being merged until your Cypress tests are green, and surface test
 results directly in your PRs with
 [pull request comments](/guides/cloud/integrations/github#Pull-request-comments)
 that include test run statistics, specific failure details, and deep links to
-results in Cypress Cloud for fast debugging. Users with
-[Business or Enterprise plans](https://www.cypress.io/pricing/) can integrate
-securely with
-[Self-managed GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/)
-instances too.
+results in Cypress Cloud for fast debugging.
 
 <DocsImage
   src="/img/dashboard/github-integration/status-checks-per-spec.jpg"
   alt="Status checks per spec"
 />
 
-### Collaborate and organize
+[Business and Enterprise plans](https://www.cypress.io/pricing/) include secure
+integration with self-managed
+[GitHub](https://docs.github.com/en/enterprise-server@3.5/admin/overview/about-github-enterprise-server)
+and [GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/) instances
+too.
 
-We also integrate with two of the world's most popular collaboration tools:
-[Slack](https://slack.com/) and [Jira](https://www.atlassian.com/software/jira).
-Deliver test results with valuable additional context, directly into a dedicated
-Slack channel. Or for those on the
-[Team, Business or Enterprise plans](https://www.cypress.io/pricing/),
-[create bidirectionally linked Jira tickets](/guides/cloud/integrations/jira#Creating-a-Jira-issue-for-a-test-case)
-directly from specific test failures.
+#### Team Communication
+
+Get notified on test progress directly in your team's
+[Slack](/guides/cloud/integrations/slack) or
+[Microsoft Teams](/guides/cloud/integrations/teams) channels.
 
 <DocsImage
   src="/img/dashboard/slack-integration/cypress-slack-integration-channel-feed.jpg"
   alt="Cypress notification feed in Slack channel"
   width={560}
 />
 
-Lastly you can use our flexible adminstrative functions to configure Cypress
-Cloud however you want, grouping [projects](/guides/cloud/projects) into
-multiple [organizations](/guides/cloud/organizations) if you have a lot,
-checking [usage](/guides/cloud/organizations#Billing-Usage), and administering
-[users and permissions](/guides/cloud/users#User-roles). We also provide
-[SSO integration](/guides/cloud/organizations#Enterprise-SSO) for teams on our
-[Enterprise plan](https://www.cypress.io/pricing/).
+#### Issue Management
 
-:::info
+Keep the whole team up-to-date with [Jira](/guides/cloud/integrations/jira)
```

Comment:
We should include a visual with the "Create Issue" icon to be consistent with the Team Communication section. But some options are subject to change here in the very near future.